cmd/module.go

```diff
@@ -65,27 +65,25 @@ func init() {
 		"The name of the module")
 }
 
-// soloBuildModuleCmd - Build module command used stand along
+// soloBuildModuleCmd - Build module command used stand alone
 var soloBuildModuleCmd = &cobra.Command{
 	Use:   "module-build",
-	Short: "Builds module",
-	Long:  "Builds module and archives its artifacts",
+	Short: "Builds module according to configurations in the MTA development descriptor (mta.yaml)",
+	Long:  "Builds module according to configurations in the MTA development descriptor (mta.yaml)",
 	Args:  cobra.NoArgs,
-	RunE: func(cmd *cobra.Command, args []string) error {
-		err := artifacts.ExecuteSoloBuild(buildModuleCmdSrc, soloBuildModuleCmdTrg, soloBuildModuleCmdExtensions, soloBuildModuleCmdModule, os.Getwd)
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Why doesn't this return an error anymore? Don't we want an indication (except in the standard output) in case it fails?

Reply:
as it is a stand alone command it doesn't have to return an error. The same we have with build and init commands. In addition if we make it returning an error it will output an error message twice. We can make silence true, but then we'll loose errors like wrong flag usage.

Reply:
All the build commands return an error (`mbt build`, `mbt project build` and `mbt module build`). I think `mbt init` should also return an error, but since we don't really recommend using it, it's less of an issue. If the output is a problem remove the `logError` call (like in `mbt build`).
If there is no indication that the module build fails this cannot be used in scripts and such (for example as part of incremental build).